assets/sass/style.sass

```diff
@@ -197,6 +197,11 @@ $colors: mergeColorMaps(("secondary": ($secondary, $white), "twitter-blue": ($tw
   .highlight
     margin-bottom: 1rem
 
+  li >
+    p:last-child:not(:only-child),
+    ul:last-child:not(:only-child)
```

Comment:
Note: In addition to addressing the two problems mentioned in #305, by using `:not(:only-child)` we ensure that lists with entries that contain a single (`p` or `ul`) child remain compact.